internal/v1/handler.go

```diff
@@ -416,6 +416,20 @@ func (h *Handlers) canUserAccessComposeId(ctx echo.Context, composeId uuid.UUID)
 	return err
 }
 
+// ConvertIgnoreImageTypeToSlice Function to convert IgnoreImageType to []string
+func ConvertIgnoreImageTypeToSlice(ignoreImageTypes *[]ImageTypes) []string {
```

Comment:
Since this is a helper function and not meant to be exposed, start with a small letter: "convertIgnoreImage..."

Reply:
done